ProcessOptimizer/tests/test_optimizer.py

```diff
@@ -440,3 +447,84 @@ def flat_score(x):
     res_noise = opt_noise.get_result()
     _, [_, res_std_reset] = expected_minimum(res_noise, return_std=True)
     assert isclose(res_std_no_white, res_std_reset, rel_tol=0.001)
+
+
+@pytest.mark.fast_test
+def test_estimate_single_x():
+    regressor = GaussianProcessRegressor(noise=0, alpha=0)
+    opt = Optimizer(
+        [(-2.0, 2.0), (-3.0, 3.0)],
+        base_estimator=regressor,
+        n_initial_points=1,
+    )
+    opt.tell([1, 1], 2)
+    X = opt.estimate([1, 1])[0]
```

Comment:
A comment on variable naming in these tests: I can't help associating X with a set of factor settings, so it messes with my reading of the code that it is an object that contains something very different. Perhaps you could name it `point_estimates`, `estimates` or `output` throughout all these tests?